### PR TITLE
fix-typo

### PR DESCRIPTION
**Pull request by hwdef**
### Ⅰ. Describe what this PR did

fix typo

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

不需要

### CODE REVIEW

File: dfget/config/metadata.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func NewMetaData(metaPath string) *MetaData {
 
 // MetaData stores meta information that should be persisted.
 type MetaData struct {
-	// ServicePort the the TCP port on which the file upload service listens.
+	// ServicePort the  TCP port on which the file upload service listens.
```
Comment (allencloud) [inline dfget/config/metadata.go]: double spaces, please remove one.

**Comment by allencloud** [comment]
Please check when you remove one word, there are only one whitespace there. @hwdef

File: docs/design/design.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ Download from the p2p network in pieces with a loop
 
 - Dfget should report the info and status of the piece downloaded when each success/fail to download a piece from the P2P network.
   - Supernode will update the bitmap information for the P2P network for better scheduling work.
-- Dfget will not download the the entire file at once but in pieces. So the Supernode will schedule several pieces of information each time for Dfget to download.
+- Dfget will not download the  entire file at once but in pieces. So the Supernode will schedule several pieces of information each time for Dfget to download.
```
Comment (allencloud) [inline docs/design/design.md]: same issue

File: supernode/daemon/mgr/cdn/file_meta_data.go [hunk 3]
```diff
@@ -183,7 +183,7 @@ func (mm *fileMetaDataManager) writePieceMD5s(ctx context.Context, taskID, fileM
 
 	// append fileMD5
 	pieceMD5s = append(pieceMD5s, fileMD5)
-	// append the the SHA-1 checksum of pieceMD5s
+	// append the  SHA-1 checksum of pieceMD5s
```
Comment (allencloud) [inline supernode/daemon/mgr/cdn/file_meta_data.go]: same issue

**Comment by hwdef** [comment]
@allencloud 
I am sorry about that.And i fixed it.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**